cmd/influx/main.go

```diff
@@ -96,6 +98,10 @@ func main() {
        Path to file to import
   -compressed
        Set to true if the import file is compressed
+  -retention-policy
```

Comment:
we we do update the parameter per my comment above, this reference would need to be updated as well